### DISCUSSION TITLE
Welcome to cycy-wrote-custom-macros discussions!

### DISCUSSION THREAD

**Discussion by cyrusfirheir** [cyrusfirheir/cycy-wrote-custom-macros#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by magnesius-collab** [comment]
The CTP macro is awesome. I have one question on it though..
With Sugarcube 236, whenever I include a `ctp` macro using `include` or `linkappend` as in the following snippet, then the content does not seem to show up.


```
<<linkappend "append">>
<<ctp "test2">>
	one
	<<ctpNext>>
	two
<</ctp>>
<</linkappend>>
```


Using the above piece of code does not display "one" upon link activation, as expected. Using the Debug mode, a <<ctp>> <</ctp>> is shown after I click the link but without any content.

**Reply by cyrusfirheir** [reply]
Should be fixed in the newest commit (f7422e3).
